ndcube/ndcube.py

```diff
@@ -592,6 +593,137 @@ def explode_along_axis(self, axis):
         # Creating a new NDCubeSequence with the result_cubes and common axis as axis
         return NDCubeSequence(result_cubes, common_axis=axis, meta=self.meta)
 
+    def pixel_axes_to_world_types(self, *axes):
```

Comment:
Add ```include_extra_coords=True``` kwarg so that users can choose to filter out ```extra_coords``` if they wish?

Thoughts @Cadair?